tests/components/google_pubsub/test_init.py

```diff
@@ -165,8 +158,8 @@ async def test_allowlist(hass: HomeAssistant, mock_client) -> None:
     ]
 
     for test in tests:
-        event = make_event(test.id)
-        handler_method(event)
+        hass.states.async_set(test.id, "not blank")
```

Comment:
Looks like we can remove the `make_event` function now.

Reply:
436ea35cf73ab203a8a31d2e0fe10ae4296c3728